content/blog/2023/2023-09-07-introduction-to-kDoc

```diff
@@ -0,0 +1,111 @@
+---
```

Comment:
This file doesn't have the `.md` extension, which is why it's not even showing up in the preview. Please fix that, push the change, and then click on the preview link in the PR to review the article ... there's lots of small things to fix.
![image](https://github.com/reflectoring/reflectoring.github.io/assets/1363578/a195b026-64fd-43c5-828c-713f931f6d28)